src/test/java/org/folio/consortia/service/UserAffiliationServiceTest.java

```diff
@@ -69,14 +70,17 @@ public void afterEach() throws Exception {
   @Test
   void primaryAffiliationAddedSuccessfullyTest() {
     var te = createTenantEntity();
+    te.setId(TENANT);
 
```

Comment:
we need to add tests that affiliation saved to central tenant also

Reply:
Test was added